packages/jest-resolve/src/shouldLoadAsEsm.ts

```diff
@@ -6,10 +6,11 @@
  */
 
 import {dirname, extname} from 'path';
+import {readFileSync} from 'fs';
```

Comment:
```suggestion
import {readFileSync} from 'graceful-fs';
```
and move one line down so it's below the `vm` import